src/main/java/de/hsh/capstoneris/bo/GroupBO.java

```diff
@@ -0,0 +1,9 @@
+package de.hsh.capstoneris.bo;
+import de.hsh.capstoneris.dao.GroupDAO;
```

Comment:
GroupDAO class is missing in this pull request, that's why the CI-check fails. Please push it to the same branch